packages/scm/src/browser/scm-provider.ts

```diff
@@ -33,6 +39,7 @@ export interface ScmProvider extends Disposable {
     readonly onDidChangeStatusBarCommands?: Event<ScmCommand[] | undefined>;
 
     readonly amendSupport?: ScmAmendSupport;
+    readonly get?: <T>(id: interfaces.ServiceIdentifier<T>) => T | undefined;
```

Comment:
it is not used by scm, it could be replaced by more specialized intereface and down casting with is operator

Reply:
that should be removed